publish-to-gts/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Publish to GtS",
```

Comment:
Can you maybe please use "GoToSocial" instead of "GtS" as name?

Reply:
Sure, will do